src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/repository/SensitiveTaskEventLogsRepository.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.cft.repository;
+
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.data.jpa.repository.Lock;
+import org.springframework.data.jpa.repository.QueryHints;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.lang.NonNull;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.reform.wataskmanagementapi.entity.SensitiveTaskEventLog;
+
+import java.util.Optional;
+import javax.persistence.LockModeType;
+import javax.persistence.QueryHint;
+
+public interface SensitiveTaskEventLogsRepository extends CrudRepository<SensitiveTaskEventLog, String>,
```

Comment:
can you move thid to the repository package in one level above. cft package is only for the search function and there is a chance to be removed as well